app/controllers/publishers_controller.rb

```diff
@@ -199,6 +203,17 @@ def update
     end
   end
 
+  def request_two_factor_authentication_removal
+    publisher = Publisher.find_by_email(params[:email])
+    if publisher
+      flash[:notice] = t("publishers.two_factor_authentication_removal.request_success")
+      MailerServices::TwoFactorAuthenticationRemovalRequestEmailer.new(publisher: publisher).perform
+    else
+      flash[:warning] = t("publishers.two_factor_authentication_removal.request_not_found")
```

Comment:
This exposes the email addresses that we have. We should change the message to the same as the above but only run `MailerServices::TwoFactorAuthenticationRemovalRequestEmailer.new(publisher: publisher).perform` if the publisher exists

Reply:
Thanks for pointing this out @yachtcaptain23 , I made the fix.